Pod/Classes/PURLogStore.m

```diff
@@ -54,10 +51,15 @@ - (instancetype)initWithDatabasePath:(NSString *)databasePath
     self = [super init];
     if (self) {
         _databasePath = databasePath;
+        _databaseReadWriteCompletionQueue = dispatch_queue_create("PureeLogStoreReadWriteCompletion", NULL);
```

Comment:
You introduce async DB readwrite #6 for avoid UI blocking.
I think better that is some BufferedOutput processes running on the background too.

Do you think that DB read write completion block should running on the main-thread?